scripts/accumulator-server.py

```diff
@@ -44,35 +43,42 @@
 import atexit
 import string
 import signal
-
-
+import argparse
```

Comment:
The following RPM package needs to be added in the .spec as dependency for contextBroker-tests: python-argparse